app/controllers/programs/managers_controller.rb

```diff
@@ -32,6 +32,7 @@ def create
         end
       else
         flash.now[:alert] = result['note']
+        @manager = Manager.new
```

Comment:
is this relevant to this PR for updating the ldap_lookup gem? If not it should be in a seprate PR

Reply:
Yes, this is relevant. The PR updates the page where you can add managers to the program. Here is the code for what to do if uniqname is not valid. I don't know why I decided to clear the field with the uniqname. I think it should not be cleared. I removed the line 35 in this commit [fe03998](https://github.com/lsa-mis/lsa_ride_share/pull/328/commits/fe03998703f4711301eb4560e43758a9741eeeac)